README.md

```diff
@@ -164,12 +164,12 @@ achieve this we must first create an instance of the Connector object.
 The constructor of the Connector class is defined as:
 
 ```
-    Connector(const std::string& broker_ws_uri,
-              const std::string& client_type,
-              const std::string& ca_crt_path,
-              const std::string& client_crt_path,
-              const std::string& client_key_path,
-              const long& connection_timeout = 5000)
```

Comment:
This is long because websocketpp uses long :( I guess I'll have to go back to longs.